docs/modules/user-guide/partials/proc_adding-container-component-to-a-devfile.adoc

```diff
@@ -2,7 +2,7 @@
 = Adding `container` components to a devfile
 
 [role="_abstract"]
-Use the `container` component type to define an image-based configuration of a container in a workspace. This allows to incorporate custom tools into the workspace. A devfile can contain one or more components of the `container` type. The `image` attribute identifies the component.
+Use the `container` component type to define an image-based configuration of a container in a workspace. This `container` component type allows you to incorporate custom tools into your workspace. A devfile can contain one or more components of the `container` type. The `image` attribute identifies the component.
```

Comment:
> The `image` attribute identifies the component.

I'm not sure which meaning identifies is supposed to be here, but it's component name that identifies each component, including container.

Reply:
Okay, I specified that the component name attribute identifies the component, not the image.